js/admin/src/utils/ExtensionRepository.js

```diff
@@ -60,4 +58,23 @@ export default class ExtensionRepository {
             m.endComputation();
         });
     }
+    toggleExtension(extension) {
+        const enabled = extension.enabled();
+
+        app.request({
+            url: app.forum.attribute('apiUrl') + '/extensions/' + id,
```

Comment:
I was wondering if we should allow enabling from our own `/bazaar/extensions/foo` endpoints. So we control the return value and can stay consistent with the id we use